### PR TITLE
Create page to convert QUICK

### PR DESCRIPTION
**Pull request by totop716**
https://app.zeplin.io/project/61661650e8d9b1b72191b00d/screen/6237f1693be1da1124cc73ec

### CODE REVIEW

File: src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            <QUICKV2Icon />
+          </Box>
+          <Typography variant='h6'>QUICK-v2</Typography>
+        </Box>
+        <Typography variant='body2' color='textSecondary'>
+          {t('convertQUICK')}
+        </Typography>
+        <Box className={classes.conversionRate}>
+          <Typography variant='caption'>
+            {t('conversionRate')}: 1 QUICK = 100 QUICK-v2
```
Comment (FriskyHamTitz) [inline src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx:133]: Isn't the conversion rate 1 to 1000

File: src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+              isInsufficientQUICK && classes.errorInput,
+            )}
+          >
+            <NumericalInput
+              placeholder='0.00'
+              value={quickAmount}
+              fontSize={18}
+              onUserInput={(value) => {
+                setQUICKAmount(value);
+                setQUICKV2Amount((Number(value) * 100).toString());
```
Comment (FriskyHamTitz) [inline src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx:152]: 100 should be 1000 and it should be stored in a const which is reused

File: src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+              }}
+            />
+            <Box
+              mr={1}
+              className={classes.maxButton}
+              onClick={() => {
+                if (quickBalance) {
+                  setQUICKAmount(quickBalance.toExact());
+                  setQUICKV2Amount(
+                    (Number(quickBalance.toExact()) * 100).toString(),
```
Comment (FriskyHamTitz) [inline src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx:162]: Same comment as above

File: src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+            {t('youwillreceive')}:
+          </Typography>
+          <Box className={classes.currencyInput}>
+            <NumericalInput
+              placeholder='0.00'
+              value={quickV2Amount}
+              fontSize={18}
+              onUserInput={(value) => {
+                setQUICKV2Amount(value);
+                setQUICKAmount((Number(value) / 100).toString());
```
Comment (FriskyHamTitz) [inline src/pages/ConvertQUICKPage/ConvertQUICKPage.tsx:191]: Same comment

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

**Review by sameepsi — APPROVED** [review]

**Review by FriskyHamTitz — APPROVED** [review]

**Pull request merged**